pkg/reconciler/jbsconfig/jbsconfig.go

```diff
@@ -529,7 +530,7 @@ func (r *ReconcilerJBSConfig) cacheDeployment(ctx context.Context, log logr.Logg
 			ValueFrom: &corev1.EnvVarSource{SecretKeyRef: &corev1.SecretKeySelector{LocalObjectReference: corev1.LocalObjectReference{Name: v1alpha1.GitSecretName}, Key: v1alpha1.GitSecretTokenKey, Optional: &trueBool}},
 		}, cache)
 
-		if jbsConfig.Spec.MavenDeployment.Repository != "" && !jbsConfig.Spec.MavenDeployment.OnlyDeploy {
+		if jbsConfig.Spec.MavenDeployment.Repository != "" {
```

Comment:
@tecarter94 This means we use the maven repository for deployment _and_ to download deployed dependencies down (rather than using OCIRepositoryClient). This allows the contaminant test to pass. It probably means `OnlyDeploy` can be removed?

Reply:
@rnc Nice one. Will be good to tidy that up.